scripts/install-ubuntu.sh

```
@@ -0,0 +1,381 @@
+if [ -d "$OSPOS_DIR" ]; then
+    echo -e "${COLOR_RED}Installation directory $OSPOS_DIR already exists${COLOR_RESET}"
+    echo -e "${COLOR_YELLOW}Remove it or set OSPOS_DIR environment variable${COLOR_RESET}"
+    exit 1
+fi
+
+echo -e "${COLOR_GREEN}[1/9] Updating system packages...${COLOR_RESET}"
+apt-get update -qq
+
+echo -e "${COLOR_GREEN}[2/9] Installing Apache, PHP, and dependencies...${COLOR_RESET}"
+# Add PHP repository for newer PHP versions if not available in default repos
+if ! apt-cache policy php${PHP_VERSION} 2>/dev/null | grep -q "Candidate:"; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)